packages/ERTP/src/paymentLedger.js

```diff
@@ -458,56 +380,127 @@ export const vivifyPaymentLedger = (
     issuerBaggage,
     name,
     assetKind,
-    getBrand,
+    brand,
+    PurseIKit,
     harden({
       depositInternal,
       withdrawInternal,
     }),
   );
 
-  const issuer = vivifySingleton(issuerBaggage, `${name} issuer`, {
-    getBrand: () => brand,
-    getAllegedName: () => name,
-    getAssetKind: () => assetKind,
-    getDisplayInfo: () => displayInfo,
-    makeEmptyPurse,
-
-    isLive,
-    getAmountOf,
-    burn,
-    claim,
-    combine,
-    split,
-    splitMany,
-  });
+  /** @type {Issuer} */
+  const issuer = vivifyFarInstance(issuerBaggage, `${name} issuer`, IssuerI, {
+    getBrand() {
+      return brand;
+    },
+    getAllegedName() {
+      return name;
+    },
+    getAssetKind() {
+      return assetKind;
+    },
+    getDisplayInfo() {
+      return displayInfo;
+    },
+    makeEmptyPurse() {
+      return makeEmptyPurse();
+    },
+    isLive(payment) {
+      // IssuerI delays calling this method until `payment` is a Remotable
+      return paymentLedger.has(payment);
+    },
+    getAmountOf(payment) {
+      // IssuerI delays calling this method until `payment` is a Remotable
+      assertLivePayment(payment);
+      return paymentLedger.get(payment);
+    },
 
-  const mint = vivifySingleton(issuerBaggage, `${name} mint`, {
-    getIssuer: () => issuer,
-    mintPayment,
+    burn(payment, optAmountShape = undefined) {
+      // IssuerI delays calling this method until `payment` is a Remotable
+      assertLivePayment(payment);
+      const paymentBalance = paymentLedger.get(payment);
+      assertAmountConsistent(paymentBalance, optAmountShape);
+      try {
+        // COMMIT POINT.
+        deletePayment(payment);
+      } catch (err) {
+        shutdownLedgerWithFailure(err);
+        throw err;
+      }
+      return paymentBalance;
+    },
+    claim(srcPayment, optAmountShape = undefined) {
+      // IssuerI delays calling this method until `srcPayment` is a Remotable
+      assertLivePayment(srcPayment);
+      const srcPaymentBalance = paymentLedger.get(srcPayment);
+      assertAmountConsistent(srcPaymentBalance, optAmountShape);
+      // Note COMMIT POINT within moveAssets.
```

Comment:
Probably. I tried once, and it was too big a hammer. We should try again. But I don't think it is relevant to this PR, so resolving. If I'm missing something, please unresolve.

The only reason it shows up in this PR is because diffs don't account well for code motion. See line 309 in the left side of the comparison.